### ISSUE TITLE
netbeans error opening project.xml

### ISSUE THREAD

**Issue by narwhal88** [samphonic/MoppyClassic#52]
hello people,
everytime i try to open the moppy java project in netbeans i get this error

Error in project.xml: Attribute name "data-pjax-transient" associated with an element type "meta" must be followed by the ' = ' character.

any help would be appreciated

**Comment by samphonic** [comment]
That's... a very unusual error to be getting.  A couple things to check:
- What version of NetBeans are you running?
- If you open up the project.xml file, does it still look like valid XML?  Especially near the "data-pjax-transient" bit.  Maybe something got corrupted in the download.  Maybe try removing just that line?  I'm not sure what it does.  =P

**Comment by narwhal88** [comment]
Just fixed the problem. Thanks for the suggestions Sam 

SOLUTION (skip to bottom paragraph for fast solution)

I re-downloaded project.xml because the way I downloaded the file was a different way than just downloading whole zip file.  The special way of downloading the project.xml didn't look like a like a proper .xml  when opened. 

So basically I downloaded the whole zip instead of the individual file and it fixed it
